x/emissions/keeper/actor_utils/losses.go

```diff
@@ -145,6 +145,84 @@ func CloseReputerNonce(
 		return err
 	}
 
+	// Calculate the regret_stdnorm and the weights (multistep process).
+	// 0. Get inferer and forecaster regrets
+	infererRegrets := regrets.InfererRegrets
+	inferers := make([]synth.Worker, 0)
```

Comment:
```suggestion
	inferers := make([]synth.Worker, 0, len(infererRegrets))
```

We can pass the capacity to make a single allocation and avoid allocation+copy when appending